src/main/java/io/kestra/plugin/surrealdb/SurrealDBConnectionInterface.java

```diff
@@ -0,0 +1,58 @@
+package io.kestra.plugin.surrealdb;
+
+import io.kestra.core.models.annotations.PluginProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Positive;
+
+public interface SurrealDBConnectionInterface {
+
+	@Schema(
+			title = "Use tls to connect"
+	)
+	@PluginProperty(dynamic = true)
```

Comment:
```suggestion
	@PluginProperty
```

We usually don't set boolean property as dynamic, it's too much work for little benefit.
Moreover, primitive (including wrapper) cannot be dynamic as the deserialization layer will not work so to have such property as dynamic you would need to define it as a String.